packages/create-block/CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 ## Unreleased
 
+### New Feature
+
+-   Adds the `npmPackages` field to the template configuration. It allows listing remote npm packages that will be installed in the scaffolded project ([#27880](https://github.com/WordPress/gutenberg/pull/27880)).
```

Comment:
Shouldn't this say npmDependencies?

Reply:
Sure it should 😅 

Fixing it now.

Reply:
Done in https://github.com/WordPress/gutenberg/commit/6d00779ffabd8926400013895819e332c8a30e9f. It might take a week or so until it's published to npm.